src/components/CustomDatagridConfigurable.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
```

Comment:
one minor change, please @salony75way 

`CustomDatagridConfigurable` doesn't quite explain what's happening.

Could we call it `DblClickDatagridConfigurable`  please?